iiify/resolver.py

```diff
@@ -387,6 +387,25 @@ def create_manifest3(identifier, domain=None, page=None):
                     #                            id=f"https://iiif.archivelab.org/iiif/{identifier}${pageCount}/canvas",
                     #                            label=f"{page['leafNum']}")
                     pageCount += 1
+    
+
+            # Setting logic for paging behavior and starting canvases
+            # Start with paged (default) or individual behaviors
+            try:
+                if bookreader['data']['brOptions']['defaults'] == "mode/1up":
+                    manifest.behavior = "individuals"
+            except:
+                manifest.behavior = "paged"
+            
+            # Then set left-to-right or right-to-left if present
```

Comment:
Add test to see if pageProgression key exists in brOptions or try catch Key not found error.